sqldelight-compiler/src/test/kotlin/app/cash/sqldelight/core/queries/PgInsertReturningTest.kt

```diff
@@ -0,0 +1,60 @@
+package app.cash.sqldelight.core.queries
+
+import app.cash.sqldelight.core.compiler.SelectQueryGenerator
+import app.cash.sqldelight.dialects.postgresql.PostgreSqlDialect
+import app.cash.sqldelight.test.util.FixtureCompiler
+import com.google.common.truth.Truth.assertThat
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+
+class PgInsertReturningTest {
+  @get:Rule
+  val tempFolder = TemporaryFolder()
+
+  @Test
+  fun `postgres INSERT RETURNING works with bind expr ?`() {
+    val file = FixtureCompiler.parseSql(
+      """
+            |CREATE TABLE data (
+            |  id INTEGER NOT NULL PRIMARY KEY,
+            |  col1 TEXT DEFAULT ''
+            |);
+            |
+            |insertReturn:
+            |INSERT INTO data
+            |VALUES ?
+            |RETURNING *;
```

Comment:
And could you add another test without `*` but specifing `RETURING data, id` to test the order?

Reply:
[2dd1ca8](https://github.com/cashapp/sqldelight/pull/3582/commits/2dd1ca885322fd7d6f1be979e3e5be5b400629ee)